railties/lib/rails/generators/app_base.rb

```diff
@@ -748,8 +748,7 @@ def run_kamal
         bundle_command "binstubs kamal"
         bundle_command "exec kamal init"
 
-        remove_file ".env"
-        template "env.erb", ".env.erb"
+        template "kamal-secrets.tt", ".kamal/secrets", force: true
```

Comment:
Should `.kamal/secrets` (or the whole `.kamal`) also be added to the [gitignore](https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/gitignore.tt) and [dockerignore](https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/dockerignore.tt) templates as well?

And potentially remove the `.env/.env.erb` lines now that there is no more an explicit template for it.

Reply:
No, .kamal/secrets is like .env.erb rather than .env. So it won't (or shouldn't!) hold actual credentials.